.github/workflows/code_check_for_flutter_vlc_player.yaml

```
@@ -0,0 +1,74 @@
+name: Code check for flutter_vlc_player folder
```

Let's stick to our naming convention:
```suggestion
name: Library ON Push & PR DO Code check
```